### PR TITLE
MEN-4533: Integration checking API compatibility with mender-connect v1.0

### PR DESCRIPTION
**Pull request by alfrunes**

### CODE REVIEW

**Comment by alfrunes** [comment]
Started a pipeline with mendersoftware/deviceconnect#67 here: https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/273346215

**Review by tranchitella — APPROVED** [review]
Wonderful

File: testutils/infra/container_manager/docker_compose_manager.py [hunk 1]
```diff
@@ -59,6 +59,12 @@ class DockerComposeNamespace(DockerNamespace):
         COMPOSE_FILES_PATH + "/storage-proxy/docker-compose.storage-proxy.yml",
         COMPOSE_FILES_PATH + "/storage-proxy/docker-compose.storage-proxy.demo.yml",
     ]
+    LEGACY_CLIENT_FILES = [
+        COMPOSE_FILES_PATH + "/docker-compose.client.yml",
+        COMPOSE_FILES_PATH + "/tests/legacy-v1-client.yml",
+        COMPOSE_FILES_PATH + "/storage-proxy/docker-compose.storage-proxy.yml",
+        COMPOSE_FILES_PATH + "/storage-proxy/docker-compose.storage-proxy.demo.yml",
+    ]
```
Comment (lluiscampos) [inline testutils/infra/container_manager/docker_compose_manager.py:67]: This is unrelated to the PR (?)
